build.gradle

```diff
@@ -163,7 +163,8 @@ subprojects {
     }
     
     tasks.withType(Test) {
-        maxParallelForks = 4
+        maxParallelForks = 1
+        ignoreFailures = true
```

Comment:
what does `ignoreFailures = true` means ?

Reply:
The intention was to let the build show all test failures, not just stop after the first failed test, but doesn't work as expected. Removing it.